### ISSUE TITLE
2025-11-19: vllm-project/vllm 仓库更新报告

### ISSUE THREAD

**Issue by yan97ao** [yan97ao/argus#436]
# 每日更新报告（2025-11-19）

## vllm-project/vllm

| 提交时间 | 作者 | 提交信息 |
|----------|------|----------|
| 2025-11-19 23:04:07 | Lucas Wilkinson | [BugFix] Fix async-scheduling + FlashAttn MLA (#28990) |
| 2025-11-19 22:13:54 | vnadathur | [torch.compile] caching of config fields should be opt-out by default (#26468) |
| 2025-11-19 22:13:50 | Yanan Cao | [CI] Reorganize compile tests so new tests are automatically included in CI (#28625) |
| 2025-11-19 21:50:30 | Harry Mellor | Relax Transformers modeling backend MoE experts check (#28952) |
| 2025-11-19 20:56:21 | Didier Durand | [Doc]: fix typos in various files (#29010) |
| 2025-11-19 20:46:24 | Chen Bruce | [Feat][Perf] Enable deepep-low-latency with round-robin expert placement. (#28449) |
| 2025-11-19 19:37:11 | Tova Movshovitz | [config] Expose `get_total_num_hidden_layers()` in ModelConfig (#28961) |
| 2025-11-19 19:32:04 | Harry Mellor | [Docs] Take env var definition out of folded admonition (#29005) |
| 2025-11-19 19:07:22 | j20120307 | [CI/Build] Fix broken build on Apple M1 (#28999) |
| 2025-11-19 19:05:44 | ihb2032 | refactor(cpu_types_scalar.hpp): Unify scalar loop implementations using unroll_loop (#28847) |
| 2025-11-19 18:35:29 | Michael Yao | [Docs] Clean up moe_kernel_features.md (#28530) |
| 2025-11-19 16:32:00 | gnovack | add support for --fully-sharded-loras in fused_moe (#28761) |
| 2025-11-19 15:47:57 | Louie Tsai | Add CPU support model (#28697) |
| 2025-11-19 14:52:30 | Didier Durand | [Doc]: fix typos in various files (#28945) |
| 2025-11-19 14:46:32 | Michael Goin | Speed up macOS smoke test (#28954) |
| 2025-11-19 14:28:40 | Roman Solomatin | [Misc] Update embedding/cross encoder tests to use `mteb` v2 (#27329) |
| 2025-11-19 13:43:01 | Lukas Geiger | [Model][QwenVL] Simplify cos/sin rotary embedding indexing  (#28962) |
| 2025-11-19 13:30:08 | Uranus | [Docs] Update oneshot imports (#28188) |
| 2025-11-19 13:21:02 | Gleb Kurchanov | [Bugfix] Fix typo in Qwen3 Next model executor (#28960) |
| 2025-11-19 13:05:22 | Xin Yang | [Bugfix] Fix FusedMoEModularKernel for triton backend (#28913) |
| 2025-11-19 12:06:21 | Matthew Bonanni | [Attention] FlashAttention ViT support, make default backend (#28763) |
| 2025-11-19 11:12:31 | Strahinja Stamenkovic | Enable bitsandbytes quantization on AMD GPUs that use warp size 32 (#27307) |
| 2025-11-19 10:32:00 | Li, Jiang | [CPU] Refactor CPU WNA16  (#28826) |
| 2025-11-19 10:10:02 | Jialin Ouyang | [Core] Switch Flat logprob control from environment variable to SamplingParams (#28914) |
| 2025-11-19 08:52:41 | Jerry Zhang | Move online quantization to `model.load_weights` (#26327) |
| 2025-11-19 08:49:36 | tomeras91 | [Hybrid][torch.compile] Refactor mamba2 forward to avoid obscuring linear projections under custom op (#28587) |
| 2025-11-19 08:45:20 | Varun Sundar Rabindranath | [Build] Add OpenAI triton_kernels (#28788) |
| 2025-11-19 08:44:27 | Johnny | [NVIDIA] Guard SM100 CUTLASS MoE macro to SM100 builds v2 (#28938) |
| 2025-11-19 04:29:06 | Michael Goin | Supress verbose logs from model_hosting_container_standards (#28949) |
| 2025-11-19 03:34:36 | Kunshang Ji | Replace `torch.cuda.Event` with `torch.Event` for better hardware compatibility (#26985) |
| 2025-11-19 03:03:23 | Chendi.Xue | [NIXL] fix cpu PD after physical <> logical block_size PR (#28904) |
| 2025-11-19 02:56:04 | Isotr0py | [Models] Replace all `nn.Conv2d` with vLLM's Conv2dLayer (#28842) |
| 2025-11-19 01:44:18 | Kevin H. Luu | [chore] Move the rest of wikimedia url to S3 (#28921) |
| 2025-11-19 01:28:34 | vllmellm | [FEAT] [AITER] [ROCm] integrate aiter sampling ops (#26084) |
| 2025-11-19 01:26:07 | Nick Hill | [Minor] Rename `ec_producer` field to `is_ec_producer` (#28884) |
| 2025-11-19 01:22:30 | Nicolò Lucchesi | [CI][NIXL] Change default `block_size` for tests (#28927) |
| 2025-11-19 00:56:29 | Luciano Martins | [Model] Add Gemma3 GGUF multimodal support (#27772) |
| 2025-11-19 00:38:22 | Ido Segev | [Benchmark] multi_turn: Report warmup-inclusive runtime (#28937) |
